### PR TITLE
complete chatapp backend

### PR DESCRIPTION
**Pull request by muskanqed**
Backend using Nodejs and PostgreSQL

### CODE REVIEW

File: controller/authController.js
Comment (rajathongal) [inline controller/authController.js:1]: write a full form of variables like req can be replaced with request and res with response, e with error. This helps other developers to understand the code

remove consoles,

Line no 35 and 45 add a status code to response and do not send the password back in any response

File: package-lock.json
Comment (rajathongal) [inline package-lock.json:1]: add this filename in gitignore and this should not be present in github

**Review by rajathongal — CHANGES_REQUESTED** [review]

File: controller/authController.js
Comment (muskanqed) [reply controller/authController.js:1]: Done

File: package-lock.json
Comment (muskanqed) [reply package-lock.json:1]: ok

**Review by rajathongal — APPROVED** [review]

**Pull request merged**